# SOME DESCRIPTIVE TITLE.
# Copyright (C) 2019, Qiskit Development Team
# This file is distributed under the same license as the Qiskit package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2019.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: Qiskit \n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2019-06-12 14:30+0000\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 2.7.0\n"

#: ../../autodoc/qiskit.providers.ibmq.api.exceptions.rst:2
msgid "qiskit.providers.ibmq.api.exceptions module"
msgstr ""

#: of qiskit.providers.ibmq.api.exceptions:1
msgid "Exceptions for IBMQ Connector."
msgstr ""

#: of qiskit.providers.ibmq.api.exceptions.ApiError:1
msgid "ベースクラス: :class:`qiskit.providers.ibmq.exceptions.IBMQError`"
msgstr ""

#: of qiskit.providers.ibmq.api.exceptions.ApiError:1
msgid "IBMQConnector API error handling base class."
msgstr ""

#: of qiskit.providers.ibmq.api.exceptions.BadBackendError:1
#: qiskit.providers.ibmq.api.exceptions.CredentialsError:1
#: qiskit.providers.ibmq.api.exceptions.RegisterSizeError:1
#: qiskit.providers.ibmq.api.exceptions.WebsocketError:1
msgid "ベースクラス: :class:`qiskit.providers.ibmq.api.exceptions.ApiError`"
msgstr ""

#: of qiskit.providers.ibmq.api.exceptions.BadBackendError:1
msgid "Unavailable backend error."
msgstr ""

#: of qiskit.providers.ibmq.api.exceptions.CredentialsError:1
msgid "Exception associated with bad server credentials."
msgstr ""

#: of qiskit.providers.ibmq.api.exceptions.RegisterSizeError:1
msgid "Exception due to exceeding the maximum number of allowed qubits."
msgstr ""

#: of qiskit.providers.ibmq.api.exceptions.WebsocketError:1
msgid "Exceptions related to websockets."
msgstr ""

#: of qiskit.providers.ibmq.api.exceptions.WebsocketTimeoutError:1
msgid "ベースクラス: :class:`qiskit.providers.ibmq.api.exceptions.WebsocketError`"
msgstr ""

#: of qiskit.providers.ibmq.api.exceptions.WebsocketTimeoutError:1
msgid "Timeout during websocket communication."
msgstr ""

